dbms/src/Interpreters/TraceLog.cpp

```diff
@@ -24,6 +24,7 @@ Block TraceLogElement::createBlock()
     {
         {std::make_shared<DataTypeDate>(),                                    "event_date"},
         {std::make_shared<DataTypeDateTime>(),                                "event_time"},
+        {std::make_shared<DataTypeUInt64>(),                                  "timestamp_ns"},
```

Comment:
But in `text_log` there is a field named `microseconds`.
For consistency, it should be named `nanoseconds` and have a type UInt32 and don't have the seconds part. But it is still acceptable, so will merge.